src/emuvim/dcemulator/node.py

```diff
@@ -280,6 +284,19 @@ def startCompute(self, name, image=None, command=None, network=None, flavor_name
         # do bookkeeping
         self.containers[name] = d
 
+        # Execute SON_EMU_CMD
```

Comment:
This should be (and as far as I know is) part of  the dummygatekeeper? No point in heaving it in the generic node module.

Reply:
yes you are correct, no idea how this ended up here...